### ISSUE TITLE
[Regression] Injustice: Gods Among Us Ultimate Edition 02.00 - Now does not work (#15429)

### ISSUE THREAD

**Issue by skolomir** [RPCS3/rpcs3#15702]
### Quick summary

Latest verstion:
rpcs3-v0.0.32-16610-caf81f13_win64
game doesn't work.

I tried to play story mode and classic battle. In last working version I finished the story mode and pass cllasic battle with few characters. In latest version and lost working version I have hanging the game when I chose story mode or classic battle.

Last working version 05-21-2024 first release:
rpcs3-v0.0.32-16500-a2dcbb9c

Link:

[](https://github.com/RPCS3/rpcs3-binaries-win/releases/download/build-a2dcbb9c13be2c0c734729d55a767a124f4ae1d9/rpcs3-v0.0.32-16500-a2dcbb9c_win64.7z)

First unworking version 05-21 - second release:
rpcs3-v0.0.32-16505-afa7a27f

Link:

[](https://github.com/RPCS3/rpcs3-binaries-win/releases/download/build-afa7a27f33a809a0acaaad80793338d4a8ed40f7/rpcs3-v0.0.32-16505-afa7a27f_win64.7z)

As you see, last working version and first unworking versiona are uploaded at the same day.

In attachment you can find rpscs.log file with name of version, and complete log information from console in both versions.

Computer specification:

I5 10 400 (6 physical/12 logical cores)
Nvidia 1650 4gb
Drivers (information from device manager): 
Date: 04-11-2024
Version: 31.0.15.5222
552.22-desktop-win10-win11-64bit-international-dch-whql.exe
16 GB ram
2TB SSD
Windows 10.
Version: 22H2 19045.4529

Hardware and sofrwave are same on both versions, i didn't update graphic driver at newer version to see will it solve the problem. I believe it will not.

I did testing with fresh instalation of rpcs and problem still existed.

Game doesn't work in the newest version of rpscx too. At this moment, the newewst version is:
rpcs3-v0.0.32-16610-caf81f13_win64

I also tried to check some libraty files in advanced menu, like libdivxdec.sprx and libvdec.sprx, but that wasn't solve the problem too.

[console - complete log file for last working version.txt](https://github.com/user-attachments/files/15811113/console.-.complete.log.file.for.last.working.version.txt)
[RPCS3 - rpcs3-v0.0.32-16500-a2dcbb9c.log](https://github.com/user-attachments/files/15811114/RPCS3.-.rpcs3-v0.0.32-16500-a2dcbb9c.log)

[console - compete log file for first unworking version.txt](https://github.com/user-attachments/files/15811109/console.-.compete.log.file.for.first.unworking.version.txt)

[RPCS3 - rpcs3-v0.0.32-16505-afa7a27f.log](https://github.com/user-attachments/files/15811115/RPCS3.-.rpcs3-v0.0.32-16505-afa7a27f.log)

### Build with regression

v0.0.32-16505-afa7a27f

### Attach two log files

[console - compete log file for first unworking version.txt](https://github.com/user-attachments/files/15811259/console.-.compete.log.file.for.first.unworking.version.txt)
[console - complete log file for last working version.txt](https://github.com/user-attachments/files/15811261/console.-.complete.log.file.for.last.working.version.txt)
[RPCS3 - rpcs3-v0.0.32-16500-a2dcbb9c.log](https://github.com/user-attachments/files/15811262/RPCS3.-.rpcs3-v0.0.32-16500-a2dcbb9c.log)
[RPCS3 - rpcs3-v0.0.32-16505-afa7a27f.log](https://github.com/user-attachments/files/15811263/RPCS3.-.rpcs3-v0.0.32-16505-afa7a27f.log)

### System configuration

I5 10 400 (6 physical/12 logical cores)
Nvidia 1650 4gb
Drivers (information from device manager): 
Date: 04-11-2024
Version: 31.0.15.5222
552.22-desktop-win10-win11-64bit-international-dch-whql.exe
16 GB ram
2TB SSD
Windows 10.
Version: 22H2 19045.4529

**Comment by Shoegzer** [comment]
This happens here too.

* OS: Linux Mint 21.3
* CPU: Intel i7-10870h
* GPU: Nvidia GTX 3070 / driver 545.29

**Comment by Asinin3** [comment]
Try latest master with RSX avcurate reservations turned on.

**Comment by skolomir** [comment]
In Advanced tab I tried "Handle RSX memory tiling", it didn't help.
For RSX FIFO accuracy, I tried Fast, Atomic,, Ordered & Atomic, it din't help too. 
Hese are the last rows of console log:

- VM: [SPU[0x3000100] Spurs w SysWorkCellSpursKernel3] Access violation reading location 0xbb (unmapped memory)
- VM: [SPU[0x1000100] Spurs w SysWorkCellSpursKernel1] Access violation reading location 0xbb (unmapped memory)
- VM: [SPU[0x0000100] Spurs w SysWorkCellSpursKernel0] Access violation reading location 0xbb (unmapped memory)
- VM: [SPU[0x2000100] Spurs w SysWorkCellSpursKernel2] Access violation reading location 0xbb (unmapped memory)
E SIG: Thread [rsx::thread] is too sleepy. Waiting for it 24865.262µs already!

In all versions which I tried after last workong version, I had error about unmapped memory. In some log I saw that communication with libdivxdec.sprx and libvdec.sprx wasn't esblished, so in firmware libraries I checket libdivxdec.sprx and libvdec.sprx. It didn't help.

I think that we can't do anything with guessing. With same setting gmae works in old version, but in new version it doesn't. So, conclusion is that settings are not problem. An exception would exist in the case that sometnig under the hood was done differently, so wee need to turn on or turn off some settings.

But, Wiki page for Injuscice 2.0 says this:

CONFIGURATION
No options that deviate from [RPCS3's default settings](https://wiki.rpcs3.net/index.php?title=Help:Default_Settings) are recommended for this title.

If wee need something to check, that would be on wiki page.
